### ISSUE TITLE
Awesome project! Question about rescale the feature length.

### ISSUE THREAD

**Issue by ideaRunner** [TencentYoutuResearch/ActionDetection-DBG#1]
Thanks for sharing!
The inference speed is extremely faster with a better result.
I'm thinking about how to reproduce the result on other datasets.
Could you share some detail or insight about how to use **linear interpolation** to   `rescale the feature length of all videos to same length 100` ?

Any method function, reference papers or codes will be thankful.

**Comment by pengxiaoxiao** [comment]
video_info_19993.json?

**Comment by pengxiaoxiao** [comment]
Hello, do you have the code to reproduce this article? The author has no working training code. How do you reason?

**Comment by lijiannuist** [comment]
@ideaRunner
After you use TSN to extract video features,  the features length is different. You must reshape these features by fitting and interpolation functions.
![image](https://user-images.githubusercontent.com/13388942/68933497-bbe20d00-07cf-11ea-90ff-7c04c07bce8f.png)

**Comment by lijiannuist** [comment]
Hi @pengxiaoxiao 
With these inference codes，you can repeat the results in our paper.

**Comment by pengxiaoxiao** [comment]
yean, thank you!

**Comment by pengxiaoxiao** [comment]
tensorflow.python.framework.errors_impl.NotFoundError: /home/shawxiao/video/ActionDetection-DBG/custom_op/prop_tcfg.so: undefined symbol: _ZN10tensorflow8internal21CheckOpMessageBuilder9NewStringEv

**Comment by pengxiaoxiao** [comment]
why?

**Comment by lijiannuist** [comment]
@pengxiaoxiao  
check your tensorflow version

**Comment by pengxiaoxiao** [comment]
Tensorflow-gpu == 1.9.0
Python == 3.6

**Comment by ideaRunner** [comment]
@lijiannuist Yes, even some video's feature-length is less than 100, it must be fixed-length to feed into the network.
Thank you for sharing such kind of straightforward code!

**Comment by ideaRunner** [comment]
@pengxiaoxiao Hi, pengxiaoxiao. For now, I am going to reproduce the inference results. 
It is better to re-open an issue as your question is not related to this one.
The more detailed information about this issue, the more accurate the solution you will get.

**Comment by pengxiaoxiao** [comment]
thank you! I solved it by g++-5!

**Comment by gtgtgt1117** [comment]
> @ideaRunner
> After you use TSN to extract video features, the features length is different. You must reshape these features by fitting and interpolation functions.
> ![image](https://user-images.githubusercontent.com/13388942/68933497-bbe20d00-07cf-11ea-90ff-7c04c07bce8f.png)

How to extract features with TSN? Is removing the FC layer of TSN a feature?

**Comment by ideaRunner** [comment]
hi @gtgtgt1117, The author answered at https://github.com/TencentYoutuResearch/ActionDetection-DBG/issues/6

**Comment by quanh1990** [comment]
> hi @gtgtgt1117, The author answered at https://github.com/TencentYoutuResearch/ActionDetection-DBG/issues/6

@ideaRunner Page missing, could you please describe the solution, thanks very much
